pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -269,6 +269,15 @@ type PoolSpec struct {
 
 	// The erasure code settings
 	ErasureCoded ErasureCodedSpec `json:"erasureCoded"`
+
+	// PoolProperties is a list of properties to enable on a given pool
+	PoolProperties []PoolProperty `json:"poolProperties,omitempty"`
```

Comment:
How about defining it as a `map[string]string`? Then it's a simple list of key-value pairs

Reply:
A bit less intuitive I think from a usability perspective (not too bad though) but easier to handle internally so ok let's go with that.